dist/tools/travis-scripts/get-pkg-list.py

```diff
@@ -29,7 +29,7 @@
 known_mcu_groups = arm_mcu_groups + msp_mcu_groups + x86_mcu_groups + \
     avr8_mcu_groups + static_tests_groups
 
-common_pkgs = ["pcregrep", "libpcre3", "python3"]
+common_pkgs = ["pcregrep", "libpcre3", "python3", "python3-pexpect"]
```

Comment:
In my experience `killpg` was the only thing that killed properly every child process (the child isn't really the problem but the children of the child, which are always generated, because `make term` generates children). Maybe storing the PID of the child before terminating it helps?